oauth2_provider/models.py

```diff
@@ -81,6 +93,7 @@ class AbstractApplication(models.Model):
 
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
+    algorithm = models.CharField(max_length=5, choices=ALGORITHM_TYPES, default=RS256_ALGORITHM)
```

Comment:
I'm not sure that `max_length=5` is a smart choice here, given that [RFC7518 Section 4.1](https://tools.ietf.org/html/rfc7518#section-4.1) has algorithm identifiers 18 characters long.

Reply:
It will store:
```
    RS256_ALGORITHM = "RS256"
    HS256_ALGORITHM = "HS256"
```